crates/ordinals/src/brc721/operations.rs

```diff
@@ -0,0 +1,24 @@
+use bitcoin::opcodes;
+
+/// The opcode used to identify register collection operations.
+pub(crate) const BRC721_INIT_CODE: opcodes::Opcode = opcodes::all::OP_PUSHNUM_15;
+
+/// Byte to identify which operation of brc721 is used. Just a fancy name to don't remember the
+/// numerical byte of each operation. Only fieldless variants are allowed, leading to a compile
+/// error otherwise.
+#[derive(Debug, Clone, PartialEq)]
```

Comment:
maybe it's better to do something like the following to fix the values ( avoids reordering broke the protocol ) :
```
#[repr(u8)] // Ensure the enum is represented as a single byte
pub(crate) enum Brc721Operation {
    RegisterCollection = 0x01,
    // TODO: Remove this attribute as soon as register ownership flow is introduced
    #[allow(dead_code)]
    RegisterOwnership = 0x02,
}
```